api/v1alpha1/dnschaos_type.go

```diff
@@ -62,6 +62,7 @@ type DNSChaosSpec struct {
 	// Action defines the specific DNS chaos action.
 	// Supported action: error, random
 	// Default action: error
+	// ui:form:enum=error;random
```

Comment:
please consider to use `// +ui:form:enum=error;random`(with prefix `+`), it would prevent this line of comment appear from crd files.

Reply:
Got it!